msgraph-teams-notification/src/msgraph/teams/notification/TeamsNotifier.java

```diff
@@ -38,7 +39,16 @@ public class TeamsNotifier extends NewTaskAssignmentListener {
   public TeamsNotifier() {
     super(IWorkflowManager.instance());
     taskHandler(this::notifyGraph);
-    client = Ivy.rest().client(UUID.fromString("fb0c277e-35a3-481f-8f79-edca67ce1145"));
+    client = backendClient();
+  }
+
+  private static WebTarget backendClient() {
+    return new MsGraph().client()
```

Comment:
// resolves the existing client from msgraph-connector; therefore we can share generated client classes and openapi inscription support.

Reply:
You can share the generated client classes. But you only can share the OPENAPI Inscription support if you use the RestClientCall Element and override the properties by hand, right?

Reply:
yes, here all ms-graph products will use the same token-store. 
- this one will be unique, as it asks for a different 'scope' + 'auth'-flow. 
- on the inscription, users could likewise set these properties, to change the token-store behavior.

still I think there will be cases where it could make sense to generate multiple clients for an easier separation: 
- e.g. if the same api is used 'in the name of a user' and on other locations with application scoped permissions.